readme.md

```diff
@@ -4,16 +4,7 @@ These samples all use Juypter Notebooks.
 
 Python requirements:
 ```
```

Comment:
You need to remove those quotes otherwise your readme renders badly :sweat_smile: